config/seata-config-core/src/main/java/io/seata/config/FileConfiguration.java

```diff
@@ -195,6 +198,7 @@ public void run() {
                     setFailResult(configFuture);
                     return;
                 }
+                LOGGER.info("The file name of the operation is {}", fileName);
```

Comment:
I think it is enough to output once at startup.

Reply:
ok, location has been modified